File: ch7 supplemental/slide4.cs
using System;
class MainClass {
  public static void Main (string[] args) {
  	
  	int ScoresLength = 5;
  	int[] scores = new int[ScoresLength];
  	
  	Console.WriteLine("Enter 5 scores");
  	
  	for(int i=0; i<ScoresLength; i++){
  		scores[i] = int.Parse(Console.ReadLine());
  	}
  	
  	Console.WriteLine("Let's print out those scores");
  	
  	int totalScore=0;
  	
  	for(int j=0; j<ScoresLength; j++){
  		Console.WriteLine("Score " + j + " :" + scores[j]);
  		totalScore+=scores[j];
  	}
  	
  	Console.WriteLine("Total score is: " + totalScore); 
  }
}
      

File: ch7 supplemental/slide4.js
var scoresLength = 5;
var scores = new Array(scoresLength);

for(var i =0; i<scoresLength;i++){
	scores[i] = prompt("enter score value")
}

console.log("Let's print out the scores we entered")

var totalScoreString = 0;
var totalScoreInt =0;

for(var i =0; i<scoresLength;i++){
	console.log(scores[i])
	totalScoreString+=scores[i];
	totalScoreInt+=parseInt(scores[i])
}

console.log("the total score is")
console.log(totalScoreString)
console.log("the real total score is")
console.log(totalScoreInt)